web/src/app/interceptors/auth.interceptor.ts

```diff
@@ -22,8 +22,8 @@ export const authInterceptor: HttpInterceptorFn = (
   }
   const accessToken = tokenMediator.getAccessToken();
   if (!accessToken) {
-    void router.navigate(['/login']);
-    return next(req);
+    // void router.navigate(['/login']);
+    // return next(req);
```

Comment:
Is this debug too ?

Reply:
Don't worry, it's because Dragos some bad things in the front caller function. I'll talk with him about that

Reply:
=(